duckduckgo/BrowserServicesKit
# Add support for GPC

**Required**:

Task/Issue URL:  https://app.asana.com/0/0/1203456821249396/f
iOS PR:  https://github.com/duckduckgo/iOS/pull/1408
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/869
What kind of version bump will this require?: Major (Breaks client code)

**Description**:
Add support for GPC

**Steps to test this PR**:
1. Ensure https://github.com/duckduckgo/privacy-reference-tests/pull/63 is merged
2. Run tests



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
